inst/rmarkdown/templates/transmissibility/skeleton/rmdchunks/EpiNow2.Rmd

```diff
@@ -17,11 +17,14 @@ library(EpiNow2)
 # will use for generation_time
 si_gamma <- epiparameter::extract_param(
   type = "range",
-  values = c(median(si$r(1e3)), min(si$r(1e3)), max(si$r(1e3))),
-  distribution = "gamma",
+  values = c(median(si$prob_dist$r(1e3)),
+             min(si$prob_dist$r(1e3)),
+             max(si$prob_dist$r(1e3))),
+  distribution = params$si_dist,
   samples = 1e3
 )
-si_gamma <- epiparameter::gamma_shapescale2meansd(si_gamma[[1]], si_gamma[[2]])
+si_gamma <- epiparameter::convert_params_to_summary_stats(
+  distribution = "gamma", shape = si_gamma[[1]], scale = si_gamma[[2]])
```

Comment:
It is not clear to me what the aim of this code is. It seems that you want to convert whatever distribution you get from `epidist_db()` into a gamma distribution. In this case I think {episoap} will load a normal distribution from `epidist_db()` and this is what needs to be converted. If my assumption is correct I think that it should be revised to either specify reading a gamma distribution with `epidist_db()` that you know is parameterised, and if that doesn't exist adding one to the {epiparameter} database. 

If my thought process is wrong please let me know what the code aims to do.

Reply:
this was to update the way that epiparameter is used since the package's functions have changed since they were first used, the point was to take distributions and convert them to gamma because at the time this is what was accepted by EpiNow2, but now this has been updated in the branch "tx_pipeline_update" to pass the pmf directly (issue #113)

Reply:
But from what I can see the `generation_time` is being passed to `epinow()` as a list and not a PMF. What am I missing?

Reply:
Are you looking at the epiparameter_update branch or tx_pipeline_update?